include/deal.II/matrix_free/operators.h

```diff
@@ -246,7 +248,7 @@ namespace MatrixFreeOperators
      * order to ensure that the cell loops will be able to access the ghost
      * indices with the correct local indices.
      */
-    void adjust_ghost_range_if_necessary(const LinearAlgebra::distributed::Vector<Number> &vec) const;
+    void adjust_ghost_range_if_necessary(const VectorType &vec) const;
```

Comment:
Do you think that this function makes sense on another vector than `LinearAlgebra::distributed::Vector`? I guess we could also move this to a stand-alone function.